platforms/android/android.toolchain.cmake

```diff
@@ -711,6 +712,14 @@ elseif( ANDROID_ABI STREQUAL "armeabi-v7a with NEON" )
  set( CMAKE_SYSTEM_PROCESSOR "armv7-a" )
  set( VFPV3 true )
  set( NEON true )
+elseif( ANDROID_ABI STREQUAL "armeabi-v7a-hard with NEON" )
+ set( ARMEABI_V7A_HARD true )
+ set( ANDROID_NDK_ABI_NAME "armeabi-v7a-hard" )
```

Comment:
btw we also need to update:
./modules/java/android_lib/jni/Application.mk 
APP_ABI := armeabi-v7a   to be APP_ABI := armeabi-v7a-hard

Maybe we need to generate APP_ABI based on what we get passed in to cmake and then I assume the install path should just work.